bindings/capacitorjs/ios/WalletFramework.xcframework/ios-x86/WalletFramework.framework/Headers/WalletFramework.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include <stddef.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+typedef struct iota_wallet_handle iota_wallet_handle_t;
+
+typedef void (*Callback)(const char* response, const char* error, void* context);
+
+extern int8_t iota_init_logger(const char* file_name, const char* level_filter);
+extern iota_wallet_handle_t* iota_initialize(const char* manager_options, char* error_buffer, size_t error_buffer_size);
+extern void iota_destroy(iota_wallet_handle_t*);
+extern void iota_send_message(iota_wallet_handle_t* wallet_handle, const char* message, Callback callback, void* context);
+extern int8_t iota_listen(iota_wallet_handle_t* wallet_handle, const char* event_types, Callback callback, void* context, char* error_buffer, size_t error_buffer_size);
+extern int8_t iota_clear_listeners(iota_wallet_handle_t* wallet_handle, const char* event_types, Callback callback, void* context, char* error_buffer, size_t error_buffer_size);
+
+
+#ifdef __cplusplus
+}
+#endif
```

Comment:
Is this file exactly the same as `bindings/capacitorjs/ios/WalletFramework.xcframework/ios-arm64/WalletFramework.framework/Headers/WalletFramework.h` and if so, can't a single file be used for both?

Reply:
You're right are the same, it's because it was created with the XCode IDE and defines that folder structure with XCFrameworks. But not sure, maybe we could use only one file, I'll investigate it, added the task https://github.com/iotaledger/wallet.rs/issues/1960